### ISSUE TITLE
Intermittent computer crashes during gameplay

### ISSUE THREAD

**Issue by TempoBR** [GTNewHorizons/GT-New-Horizons-Modpack#15278]
### Your GTNH Discord Username

tempo9055

### Your Pack Version

2.5.1

### Your Server

SP

### Java Version

Java 17

### Type of Server

Single Player

### Your Actions

 Computer freezes up after approx. 30-45 minutes of GTNH, and either resets without showing a BSOD with the instance producing a crash report, or shows a BSOD and no crash report. I migrated from Curseforge to Prism Launcher with the same save, and it has continued. No gameplay action triggers the crash as far as I can tell, it seems to happen when mining most often, but has happened other times.

### Crash Report

[crash-2024-01-05_19.19.11-server.txt](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/files/13848412/crash-2024-01-05_19.19.11-server.txt)
[crash-2024-01-05_19.45.32-server.txt](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/files/13848413/crash-2024-01-05_19.45.32-server.txt)

These are only the times it's produced a crash report, most often it does not.

[fml-client-latest.log](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/files/13848415/fml-client-latest.log)

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the crash still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this crash consistently by following the exact steps I described above.
- [ ] I have asked other people and they confirm they also crash by following the exact steps I described above.

**Comment by bombcar** [comment]
```
java.lang.ArrayIndexOutOfBoundsException: Index -4 out of bounds for length 2048
    at net.minecraft.world.chunk.NibbleArray.get(SourceFile:28)
    at net.minecraft.world.chunk.storage.ExtendedBlockStorage.getExtBlockMetadata(SourceFile:75)
    at net.minecraft.world.chunk.Chunk.getBlockMetadata(Chunk.java:548)
    at net.minecraft.world.chunk.Chunk.func_150812_a(Chunk.java:878)
    at net.minecraft.world.chunk.Chunk.addTileEntity(Chunk.java:862)
    at net.minecraft.world.chunk.storage.AnvilChunkLoader.loadEntities(AnvilChunkLoader.java:500)
    at net.minecraftforge.common.chunkio.ChunkIOProvider.callStage2(ChunkIOProvider.java:41)
    at net.minecraftforge.common.chunkio.ChunkIOProvider.callStage2(ChunkIOProvider.java:12)
    at net.minecraftforge.common.util.AsynchronousExecutor.skipQueue(AsynchronousExecutor.java:344)
    at net.minecraftforge.common.util.AsynchronousExecutor.getSkipQueue(AsynchronousExecutor.java:302)
    at net.minecraftforge.common.chunkio.ChunkIOExecutor.syncChunkLoad(ChunkIOExecutor.java:12)
    at net.minecraft.world.gen.ChunkProviderServer.loadChunk(ChunkProviderServer.java:126)
    at net.minecraft.world.gen.ChunkProviderServer.loadChunk(ChunkProviderServer.java:101)
    at net.minecraft.server.MinecraftServer.initialWorldChunkLoad(MinecraftServer.java:265)
    at net.minecraft.server.integrated.IntegratedServer.loadAllWorlds(IntegratedServer.java:78)
    at net.minecraft.server.integrated.IntegratedServer.startServer(IntegratedServer.java:92)
    at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:387)
    at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)
```

This is one I've not seen before. Can you run a RAM test on your machine just to verify it's not bad memory?

**Comment by TempoBR** [comment]
I just ran MDsched.exe, the computer detected no memory errors, lmk if you need a different test. 
I have 2 sticks of 16gb memory clocked at 1600 MHz, DDR4. I was originally running GTNH with 16gb of dedicated memory on Curseforge when I first encountered the crashes, I turned it down to 8gb after some research, but they continued. I am still at 8gb of dedicated memory after migrating to Prism.

**Comment by bombcar** [comment]
Graphics card?

does it happen if you play a new world for 40m? Or in your existing world leave your base for an hour?

**Comment by TempoBR** [comment]
Graphics card is a 3060 12gb, I don't think it's being taxed too hard but I could be wrong. I've only tested it with my transferred saved world, I'll get back to you on that. It's crashed when I was both in my base and away from it for at least two hours; I'm in middle-early steam so if that's running my hardware too high I'd be shocked. I checked thermals on at least one of the crashes and everything was well under anything scary, though I couldn't measure it while it froze.
